### DISCUSSION TITLE
Rate Component

### DISCUSSION THREAD

**Discussion by knpwrs** [mantinedev/mantine#2079] [Feature requests]
A Rate component like the one from Element Plus would be really cool to have in Mantine: https://element-plus.org/en-US/component/rate.html

The interactions in the component from Element Plus in particular feel really nice.

<img width="137" alt="image" src="https://user-images.githubusercontent.com/174864/184189934-41dd2e75-7fac-4163-aa57-bbcaea3a6735.png">

<img width="144" alt="image" src="https://user-images.githubusercontent.com/174864/184190011-8a2906e0-1a81-424c-80d7-77a371b8cc98.png">

**Comment by rtivital** [comment]
Rating component is in the backlog, it will be added in one of the next releases
